router/core/websocket.go

```diff
@@ -664,7 +670,7 @@ type WebSocketConnectionHandler struct {
 	request    *http.Request
 	conn       *wsConnectionWrapper
 	protocol   wsproto.Proto
-	clientInfo *ClientInfo
+	detailedClientInfo clientinfo.DetailedClientInfo
```

Comment:
Should we leave this field name as `clientInfo`? Then with the interface as `client.Info`.